modules/gcloud/bigquery_test.go

```diff
@@ -81,8 +88,84 @@ func ExampleRunBigQueryContainer() {
 		}
 	}
 
-	fmt.Println(val)
-
+	fmt.Println(val[0])
```

Comment:
Question: could this change lead to a panic if no result was returned?

Reply:
It's a testable example where we expect it will return the error. If for some reason any of the BigQuery impl iror the client change, we will be notified by the failure. But let me revert it, as it's harmless in both directions